platform-emf/src/test/java/com/softicar/platform/emf/attribute/field/foreign/entity/input/EmfEntityInputTest.java

```diff
@@ -33,6 +35,14 @@ public void testGetValueWithFirstName() {
 		assertSame(user, getInputValue());
 	}
 
+	@Test
+	public void testGetValueWithBrowsePopover() {
+
+		openBrowsePopover();
+		findNode(DomPopover.class).clickNode(user.toDisplayWithoutId());
```

Comment:
```suggestion
		findNode(DomPopover.class).clickNode(user.toDisplay());
```
...i think this needs to be modified as well.